doc/rbd/rados-rbd-cmds.rst

```diff
@@ -118,24 +118,39 @@ For example::
 Retrieving Image Information
 ============================
 
-To retrieve information from a particular image, execute the following,
-but replace ``{image-name}`` with the name for the image:: 
+To retrieve information from a particular image, run the following command, but
+replace ``{image-name}`` with the name for the image:
 
-	rbd info {image-name}
+.. prompt:: bash $
+
+   rbd info {image-name}
 	
-For example::
+For example:
 
-	rbd info foo
+.. prompt:: bash $
+
+   rbd info foo
 	
-To retrieve information from an image within a pool, execute the following,
-but replace ``{image-name}`` with the name of the image and replace ``{pool-name}``
-with the name of the pool:: 
+To retrieve information from an image within a pool, run the following command,
+but replace ``{image-name}`` with the name of the image and replace
+``{pool-name}`` with the name of the pool:
 
-	rbd info {pool-name}/{image-name}
+.. prompt:: bash $
 
-For example:: 
+   rbd info {pool-name}/{image-name}
+
+For example:
+
+.. prompt:: bash $
+
+   rbd info swimmingpool/bar
 
-	rbd info swimmingpool/bar
+.. note:: Other naming conventions are possible, and might conflict with the
+   naming convention described here. For example, ``userid/<uuid>`` is a
+   possible name for an RBD image, and such a name might (at the least) be
+   confusing. In the intrest of helping the reader of this documentation to
+   form a clear idea of the way that RBD images are named, we offer this
+   syntax: ``rbd -p {pool-name} | info {image-name}``.
```

Comment:
Where did `|` came from?  It's going to be interpreted as a pipe to `info` by the shell...

Reply:
Also, might have been useful to link "Image, snap, group and journal specs" section of the man page (https://docs.ceph.com/en/latest/man/8/rbd/#image-snap-group-and-journal-specs) here.

Reply:
> Where did `|` come from? It's going to be interpreted as a pipe to `info` by the shell...

It's true. It's true.  This is my fault. I'll remove it in a follow-up PR.

Reply:
> Also, might have been useful to link "Image, snap, group and journal specs" section of the man page (https://docs.ceph.com/en/latest/man/8/rbd/#image-snap-group-and-journal-specs) here.

I'll link to this in a future PR. One of the reasons that we have developed this accidentally irritating workflow is so that we can ingest suggestions like this.

As I said (or at least meant to) in another comment on this page, I'm sorry that our docs workflow spammed your inbox and made your workweek crummier than it otherwise would have been.